packages/allow-scripts/src/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "../../../.config/tsconfig.src.json",
+  "compilerOptions": {
+    "outDir": "../types",
+    "paths": {
+      "@lavamoat/aa": ["../../aa"]
+    }
+  },
+  "include": ["."],
+  "references": [{ "path": "../../aa/tsconfig.json" }]
```

Comment:
Why do we need references? I thought I know what they're for, but then Leo told me everything works the same with them removed...

Reply:
It might work the same if you're lucky.  The references basically tell `tsc`'s incremental builder "compile x before y".  This is especially important when running `tsc --watch`, but if you omit them, you can get into weird situations where some workspace either hasn't been compiled yet or it's providing old types to its consumer.  I have suffered.

Reply:
cc @leotm